.github/workflows/tests.yml

```diff
@@ -18,10 +18,12 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-              php-version: ['7.3', '7.4', '8.0']
-              db-image: ['mysql:5.7'] # a docker-hub key
               include:
+                  - php-version: '7.3'
+                    db-image: 'mysql:5.6'
                   - php-version: '7.4'
+                    db-image: 'mysql:5.7'
+                  - php-version: '8.0'
                     db-image: 'mysql:8.0'
```

Comment:
@staabm Was hältst du davon, nur diese Kombis zu testen?
MySQL 5.6 hatten wir vorher gar nicht. 
So haben wir jetzt 5.6, 5.7 und 8.0, aber jeweils nur mit einer PHP-Version, also nicht kreuz und quer.
Ich glaube, mir würde das so aber reichen.
(Plus die zwei MariaDB-Tests, die bleiben so.)

Reply:
was ist die motivation diese zu reduzieren?

prinzipiell bin ich nicht verheiratet mit dem bisherigen setup

Reply:
also vorher hatten wir (mariadb außen vor, da nicht geändert):
- mysql 5.7 mit php 7.3, 7.4, 8.0
- mysql 8 mit php 7.4

jetzt möchte ich mysql 5.6 noch testen. jetzt könnte man noch weitere ergänzen, aber mir kommt es unnötig vor, kreuz und quer zu verknüpfen. halte es für unwahrscheinlich genug, dass es da fehler nur in den fehlenden kombis gibt.
deswegen eben mein vorschlag:
- mysql 5.6 mit php 7.3
- mysql 5.7 mit php 7.4
- mysql 8.0 mit php 8.0

(also ein build weniger als vorher, neu aber auch mysql 5.6 dabei)

Reply:
oder anders gesagt: die frage ist, warum wir gerade mysql 5.7 vorher mit allen php-versionen verknüpft haben, mysql 8 aber nur mit einer. und was wir dann mit mysql 5.6 machen würden, mit allen oder auch nur einer.